frontend/src/components/Comments.jsx

```diff
@@ -0,0 +1,59 @@
+import React , {useState, useRef}from 'react';
+import { Typography, TextField, Button} from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { mergeClasses } from '@material-ui/styles';
```

Comment:
mergeClasses is imported from matrial-ui, but never being used. Please remove it.